test_site/pages/section_experiments.rb

```diff
@@ -33,7 +33,8 @@ def downcase_title_text
 
   def cell_value=(value)
     execute_script(
-      "document.getElementById('first_search_result').children[0].innerHTML = '#{value}'"
+      "document.getElementById('first_search_result').children[0].innerHTML =
```

Comment:
Github uses a ~120 character limit on a 15" MBP; setting your limit to 120 would probably resolve these issues and potentially keep things more easily readable.

Reply:
I have a "rough" plan of revisiting and tidying up all of the sample POM and JS / sample pages. One thing we can do is remove some duplication in the naming of things so we don't hit an issue where we get these stupidly long lines!